modules/dedicated-creating-your-cluster.adoc

```diff
@@ -5,7 +5,9 @@
 [id="dedicated-creating-your-cluster_{context}"]
 = Creating your cluster
 
-To create your {product-title} cluster:
+Use the following steps to create your {product-title} cluster.
```

Comment:
Just noting that I was unable to find this file in a preview (even a local build in the dedicated distro) to verify that this section looks okay.